packages/cydran/src/log/appender/ConsoleAppenderImpl.ts

```diff
@@ -1,166 +1,112 @@
-};
+const preamOrdrKey: string = PropertyKeys.CYDRAN_LOG_PREAMBLE_ORDER as const;
+
+const PREAMBLFMT: string = "%c%s " as const;
 
 type OutColor = {orig: string, alt: string};
+type LogPrep = {color: string, preamble: string};
 
-class ConsoleAppenderImpl implements Appender {
+class ConsoleAppender extends AbstractAppender {
 
 	private preambleOrder: string[];
 
 	private wkColors: SimpleMap<OutColor> = {
+		FATAL: {orig: "#ff2f92", alt: null},
+		ERROR: {orig: "#ff2f92", alt: null},
 		WARN: {orig: '#ff9400', alt: null},
-		TRACE: {orig: "#ffd478", alt: null},
-		FULLSTACK: {orig: "#ff2f92", alt: null},
+		INFO: {orig: "#0096ff", alt: null},
 		DEBUG: {orig: "#008e00", alt: null},
-		INFO: {orig: "#0096ff", alt: null}
+		TRACE: {orig: "#ffd478", alt: null}
 	};
 
-	private static readonly id: string = "default";
-
 	private console: Console;
 
-	public constructor(props?: Properties, consoleInstance?: Console) {
-		this.console = defaulted(consoleInstance, console);
-		this.preambleOrder = "time:level:name".split(":");
-		this.setPreferences(props);
+	public constructor(id: string, pOrder: string = "level:time:name", wkConsole: Console = console) {
+		super(id);
+		this.console = wkConsole;
+		this.preambleOrder = pOrder.toLowerCase().split(":");
 	}
 
-	public trace(name: string, payload: any, error?: Error): void {
-		const stacked: Error | boolean = isDefined(error) ? error : false;
-		const preamble: string = this.createPreamble(name, "TRACE");
-		const stackedIsErr: boolean = (stacked instanceof Error);
-		const printFullStack: boolean = !stackedIsErr && !!stacked;
-		const color: string = (printFullStack) ? this.wkColors.FULLSTACK.alt || this.wkColors.FULLSTACK.orig: this.getColor("TRACE");
-		this.console.log(`%c${ preamble }`, `color:${ color }`, payload);
-	}
-
-	public debug(name: string, payload: any, error?: Error): void {
-		const stacked: Error | boolean = isDefined(error) ? error : false;
-		const preamble: string = this.createPreamble(name, "DEBUG");
-		const stackedIsErr: boolean = (stacked instanceof Error);
-		const printFullStack: boolean = !stackedIsErr && !!stacked;
-		const color: string = (printFullStack) ? this.wkColors.FULLSTACK.alt || this.wkColors.FULLSTACK.orig: this.getColor("DEBUG");
-		this.console.log(`%c${ preamble }`, `color:${ color }`, payload);
-	}
-
-	public info(name: string, payload: any, error?: Error): void {
-		const stacked: Error | boolean = isDefined(error) ? error : false;
-		const preamble: string = this.createPreamble(name, "INFO");
-		const stackedIsErr: boolean = (stacked instanceof Error);
-		const printFullStack: boolean = !stackedIsErr && !!stacked;
-		const color: string = (printFullStack) ? this.wkColors.FULLSTACK.alt || this.wkColors.FULLSTACK.orig: this.getColor("INFO");
-		this.console.log(`%c${ preamble }`, `color:${ color }`, payload);
+	public log(lvl: Level, label: string, msg: string, wkArgs?: any[], error?: Error) {
```

Comment:
Wouldn't this method be private?

Reply:
nope... not part of public interface (Appender)